src/python/pants/option/config_test.py

```diff
@@ -98,7 +175,10 @@ def setUp(self) -> None:
         "movie": "inception",
       },
     }
-    self.expected_file2_options: Dict[str, Dict[str, str]] = {
+
+  @property
+  def expected_file2_options(self):
```

Comment:
Having these defined closer to the actual files would be great I think. Maybe even

```
class File1:
  INI_CONTENTS = """...""" 
  TOML_CONTENTS="""..."""
  
  @property
  def expected_options(self): ...
```

I think this would help people modifying the tests later.

Reply:
Having read the rest of the file, I'm even more a fan of this approach.